### PR TITLE
Vehicle Data

### PR DESCRIPTION
**Pull request by khakers**
Explanations of Vehicle properties similar to [ItemData](https://github.com/SmartlyDressedGames/U3-Docs/blob/master/ItemData.md). Organization and explanations are subject to change based on the final distribution and recommendations are welcome.

### CODE REVIEW

File: VehicleData.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+__BatteryMode\_Headlights__: `Charge`, `Burn`, `None`.
+
+__BatteryMode\_Sirens__: `Charge`, `Burn`, `None`.
+
+__Fuel\_Burn\_Rate__: The rate fuel burns at. Set to 2.05 for Car, 4.2 for others by default.
+
+__Pitch\_Idle__: Changes the pitch of the engine audio at idle. If your audio clip is named Engine_Large or Engine_Medium this is preset to .75 and .625
+
+__Pitch\_Drive__: Changes the pitch of the engine audio while driving. If your audio clip is named Engine_Large or Engine_Medium this is preset to .075 and .025
+
```
Comment (MoltonMontro) [inline VehicleData.md:69]: For the lines from "Speed_Min" to "Passenger_Explosion_Armor", these are not Battery-related.

File: VehicleData.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+__Fuel\_Min__: Minimum fuel that the vehicle can spawn with.
+
+__Fuel\_Max__: Maximum fuel that the vehicle can spawn with.
+
+__Fuel__: Maximum ammout of fuel the vehicle can hold.
+
+__Health\_Min__: Minimum health that the vehicle can spawn with.
+
+__Health\_Min__: Maximum health that the vehicle can spawn with.
```
Comment (MoltonMontro) [inline VehicleData.md:88]: **Health_Min** is listed twice, and **Health** is missing (this is the default spawning health).

File: VehicleData.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+__Explosion__: ID of the explosion effect to use when destroyed.
+
+__Explosion_Min_Force_Y__: Defaults to `00, 1024, 0.0`
+
+__Explosion_Max_Force_Y__: Defaults to `00, 1024, 0.0`
+
```
Comment (MoltonMontro) [inline VehicleData.md:95]: Missing SqrDelta

File: VehicleData.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+__ID__: The item ID is used to spawn the item into the game, and is represented as an unsigned 16 bit integer (a range of 0–65535). It is recommended not to use a value less than 2,000 as those are reserved for official content. It is also recommended to avoid any ID range being used by curated content, as those are often used by modded servers and custom Workshop maps. Vehicles ID ranges do not conflict with those of Items or Objects.
+
+__Bypass\_ID\_Limit__: Allows you to use an ID that is within the space reserved for vanilla content.
+
+__Size2\_Z__: Controls orthagonal camera size for vehicle skin icons. Basically irrelevant since very few vehicles have skins.
+
+__Shared_Skin_Name__:
+
+__Shared_Skin_Lookup_ID__ `uint16`
```
Comment (MoltonMontro) [inline VehicleData.md:26]: For context, the lookup ID is the ID of the base vehicle being referenced for skins, which is also the vehicle that has the skin context type (e.g., APC).

File: VehicleData.md (hunk 2 above)
Comment (khakers) [reply VehicleData.md]: Can't believe I missed **Health** 🤦. Fixed these issues in 127de6c3af546622c60cc7189097a9384bf55aa4.

**Pull request closed without merging**